android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/dashboard/DashboardFragment.java

```diff
@@ -0,0 +1,88 @@
+package de.dhbw.ka.se.fibo.ui.dashboard;
+
+import android.os.Bundle;
+import android.text.format.DateFormat;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.core.util.Pair;
+import androidx.fragment.app.Fragment;
+import androidx.lifecycle.ViewModelProvider;
+
+import com.google.android.material.datepicker.MaterialDatePicker;
+
+import java.util.Date;
+
+import de.dhbw.ka.se.fibo.R;
+import de.dhbw.ka.se.fibo.databinding.FragmentDashboardBinding;
+
+public class DashboardFragment extends Fragment {
+
+    private FragmentDashboardBinding binding;
+    private MaterialDatePicker<Pair<Long, Long>> picker;
+    private Date startDate;
+    private Date endDate;
+
+
+    public View onCreateView(@NonNull LayoutInflater inflater,
+                             ViewGroup container, Bundle savedInstanceState) {
+        DashboardViewModel dashboardViewModel =
+                new ViewModelProvider(this).get(DashboardViewModel.class);
+
+        binding = FragmentDashboardBinding.inflate(inflater, container, false);
+        View root = binding.getRoot();
+
+
+
+        initializeDateCard();
+        createDatePicker();
+
+
+        return root;
+    }
+
+    private void initializeDateCard() {
+        setDateCardTitle();
+        setDateCardTime();
+        setListener();
+    }
+
+    private void setListener() {
+        binding.button.setOnClickListener(e -> picker.show(requireActivity().getSupportFragmentManager(), "date_pange_picker"));
+    }
+
+    private void setDateCardTime() {
+        if (startDate == null) {
+            startDate = new Date();
+            startDate.setMonth(startDate.getMonth() - 1);
+        }
+        if (endDate == null) {
+            endDate = new Date();
+        }
+
+        binding.dateStartEndText.setText(String.format("%s - %s", DateFormat.format("dd.MM", startDate), DateFormat.format("dd.MM", endDate)));
+    }
+
+    private void setDateCardTitle() {
+        binding.datePickerTitle.setText(R.string.dateCardTitle);
+    }
+
+    private void createDatePicker() {
+        picker = MaterialDatePicker.Builder
+                .dateRangePicker()
+                .setTitleText(R.string.datePickerTitle)
+                .build();
```

Comment:
I don't know how to fix this, but maybe we can enforce the locale to Germany, so everything is localized:
<img width="405" alt="image" src="https://user-images.githubusercontent.com/11234139/201598106-5f4b2d64-87fe-44b9-a327-75a8841ab3cd.png">


I know this is because I have set everything to English on my local machine, nonetheless we should always use the German locale (because we only support that language for now).